index.html

```diff
@@ -0,0 +1,132 @@
+            </ul>
+        </nav>
+
+    </header>
+
+    <section class="intro">
+        <p id="hello">"Hello!,Sharing world"</p>
+        <div class="title-intro">
+            <h1>FORMULA 1 FANS GLOBAL SUMMIT 2023</h1>
+        </div>
+        <div class="p-intro">
+            <p class="summit-intro">Welcome to the 7th global summit of Formula 1 fans.
+                Celebrate,have fun and bless someone.Venue:Vienna,Austria. </p>
+        </div>
+        <div class="date">
+            <p class="show-date">2023.9.8(FRI) ~ 9(SAT)</p>
+            <p>@Grand Prix of Vienna and more</p>
+        </div>
+    </section>
+
+    <section class="program">
+        <h2>Main program</h2>
+        <hr class="sm-line">
+
+        <div id="gr-program">
+            <div class="grid-program">
+                <img src="images/program_icon_01 1.svg" alt="lecture-icon" class="program-icon">
+                <p class="program-points">Lecture</p>
+                <p class="program-ex">Listen to the speakers from various countries 
+                    about the messages of sharing and opening.</p>
+            </div>
+
+            <div class="grid-program">
+                <img src="images/program_icon_02 1.svg" alt="exhibition-icon" class="program-icon">
+                <p class="program-points">CC Exhibition</p>
+                <p class="program-ex">Appreciate various creations applying 
+                    CC license of artists, organized from Art Center Nabi</p>
+            </div>
+
+            <div class="grid-program">
+                <img src="images/program_icon_03 1.svg" alt="forum-icon" class="program-icon">
+                <p class="program-points">Forum</p>
+                <p class="program-ex">Have the time to share your thoughts and 
+                    opinions with experts for each topic.</p>
+            </div>
+
+            <div class="grid-program">
+                <img src="images/program_icon_04 1.svg" alt="workshop-icon" class="program-icon">
+                <p class="program-points">Workshop</p>
+                <p class="program-ex">Try creating your own work using open source license 
+                    rather than just watching at it.</p>
+            </div>
+
+            <div class="grid-program">
+                <img src="images/program_icon_05 1.svg" alt="ignite-icon" class="program-icon">
+                <p class="program-points">CC ignite</p>
+                <p class="program-ex">get opportunities to network with CC affiliates around the world, 
+                    also after the summit.</p>
+            </div>
+        </div>
+
+        <p id="see-program1">SEE THE WHOLE PROGRAM</p>
+        <button class="btn-summit">Join Global Summit 2023</button>
+ 
+    </section>
+
+    <section>
+        <h2 class="feature">Feature Speakers</h2>
+        <hr class="feature-dash">
+        <div class="summit-speaker">
+            <div id="speaker-card-container"></div>
+        </div>
+    </section>
```

Comment:
- [ ] Make sure that the speaker section is centered, the alignment should be exactly as it is in the Cindly design

- [ ] It could be better if background behind the image is present in your design 👍 

### Your design
![Capture44](https://github.com/MarkYegon7/Capstone-Project/assets/80458199/8b726f22-f96f-4bd1-8ad6-c0ca558d008c)
### Cindly design 
![Capture444](https://github.com/MarkYegon7/Capstone-Project/assets/80458199/767339be-dc9d-41a1-8441-3d851e0dcac1)

Reply:
Centered the speaker section and added background color